pkg/kubernetes/scripts/virtual-machines/components/VmsListing.jsx

```diff
@@ -19,33 +19,36 @@
 
 // @flow
 
-import React from 'react'
-import { connect } from 'react-redux'
-import { gettext as _ } from 'cockpit'
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { gettext as _ } from 'cockpit';
 
-import { Listing } from '../../../../lib/cockpit-components-listing.jsx'
-import VmsListingRow from './VmsListingRow.jsx'
+import { Listing } from '../../../../lib/cockpit-components-listing.jsx';
+import VmsListingRow from './VmsListingRow.jsx';
+
+React;
```

Comment:
Curious, how come the linter didn't complain about the unused React before?

Reply:
Before these changes, the `React` was used in `propTypes` bellow. I changed to import `PropTypes`, so we are consistent across various source files. We can expect move towards `React` from `react-lite` in the future, so the `PropTypes` will be taken from separate npm package. This consistency will help with the bulk-update of sources.